### PR TITLE
Added sample of how to Find-and Replace text inside table in Word document

### PR DESCRIPTION
**Pull request by elizabethatienoo**
Added sample to find and replace text inside table in Word document

### CODE REVIEW

File: Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        static void Main(string[] args)
+        {
+            Program program = new Program();  
+            //Load the Input document.
+            using (WordDocument document = new WordDocument(Path.GetFullPath(@"../../Input.docx"), FormatType.Docx))
+            {              
+                //Access table in Word document.
+                WTable table = document.Sections[0].Tables[0] as WTable;
+                program.FindTextWithinTable(table, document);
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs:18]: Whether we need to call using class name?

File: Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                //Save the document.
+                document.Save(Path.GetFullPath("../../Sample.docx"), FormatType.Docx);
+            }
+        }
+        /// <summary>
+        /// Find text within table in Word document.
+        /// </summary>
+        /// <param name="table"></param>
+        /// <param name="paragraph"></param>
+        public void FindTextWithinTable(WTable table, WordDocument document)
```
Comment (MohanaselvamJothi) [inline Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs:28]: Change method name as private void FindAndReplaceInTable()

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs (hunk 1 above)
Comment (elizabethatienoo) [reply Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs]: No, I have changed the method access and naming to correct this.

File: Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs (hunk 2 above)
Comment (elizabethatienoo) [reply Find-and-Replace/Find-and-replace-text-within-table/.NET-Framework/Find-and-replace-text-within-table/Program.cs]: This is addressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**